src/mpol/datasets.py

```diff
@@ -126,7 +155,7 @@ def to(self, device):
 
 
 # custom dataset loader
-class UVDataset(Dataset):
+class UVDataset(torch_ud.Dataset):
```

Comment:
I think this class is stale and we'll want to eventually rewrite it, using some of the ideas put forth in #126. But for now it can come along for the ride.

Reply:
You are right -- it is stale. I searched for references in the code base but there were none. I thought I'd keep it there in case I missed it.